scripts/tax-id-analysis/plot-data.py

```diff
@@ -0,0 +1,39 @@
+import json
```

Comment:
Wrote a separate script to further process the data output from `checkDuplicateTaxIds.ts`. Could have been done in the same script file, but this was easier.